### PR TITLE
Adding Scoreboard command to the info.lua

### PR DESCRIPTION
**Pull request by tonitch**
Implementing /scoreboard as part of the #158 
info.lua updated for an overview of the work to do

### CODE REVIEW

**Comment by tonitch** [comment]
I just noticed that my ide removed all the indentation. I don't know if this is a problem

**Comment by 12xx12** [comment]
Hi and welcome to the project.

About the whitespace:
The change itself is not that bad. But to keep this PR clean, pls move it to a new PR

**Comment by tonitch** [comment]
This is basically it for the command, I will add Comments and more comprehensive error message then It will be good to merge I think

**Comment by bearbin** [comment]
I'll review this tomorrow in depth. It looks good generally, but the documentation messages could do with some standardisation, and the commands could do with some changes to correspond better to the vanilla ones.

**Comment by tonitch** [comment]
I used [this](https://minecraft.fandom.com/wiki/Commands/scoreboard) as reference for the commands, I tried to be as accurate as possible with the commands but there was not the outputs of the commands so I made it a bit like I though

I will launch a minecraft world to test and documents the outputs more accurately, I just have no time this w-e, I will try to do this next week

**Comment by tonitch** [comment]
I want to make sure the code is clean and follow the project's structure so I checked other files as [cmd_effect.lua](https://github.com/cuberite/Core/blob/master/cmd_effect.lua) for instance.

But I don't understand, Why does the [Response Variable](https://github.com/cuberite/Core/blob/master/cmd_effect.lua) is [returned](https://github.com/cuberite/Core/blob/master/cmd_effect.lua) as this doesn't seem to be handle by cuberite but also, why should I return the content of [cPlayer:sendMessage to Response](https://github.com/cuberite/Core/blob/master/cmd_effect.lua) as this function doesn't seems to return anything...

**Comment by mathiascode** [comment]
> Why does the [Response Variable](https://github.com/cuberite/Core/blob/master/cmd_effect.lua) is [returned](https://github.com/cuberite/Core/blob/master/cmd_effect.lua) as this doesn't seem to be handle by cuberite

It's used for the console command to print the response: https://github.com/cuberite/Core/blob/baf6a4e7050f8ed3c19dab50924863ed704a3eba/cmd_effect.lua#L67-L69

> but also, why should I return the content of [cPlayer:sendMessage to Response](https://github.com/cuberite/Core/blob/master/cmd_effect.lua) as this function doesn't seems to return anything...

There are additional functions that return the message when appropriate: https://github.com/cuberite/Core/blob/baf6a4e7050f8ed3c19dab50924863ed704a3eba/core_functions.lua#L72-L97

**Comment by tonitch** [comment]
I am currently implementing the console side of the command but I have a problem... scoreboard by cuberite work as a per world basis. so when a player execute a scoreboard command I usually get the player's world to create the scoreboard... but when the console does it. what should I do?

File: cmd_scoreboard.lua [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+
+function resetAllObjectives(Objective)
+	Objetive:ResetScore(tplayer)
+end
+
+function sendListObjectives(Objective)
+	if gPlayer then
+		gPlayer:SendMessage(Objective:GetDisplayName() .. " -> " .. Objective:GetName() .. ": " .. get_key_for_value(criterias, Objective:GetType()))
+	else
+		LOG(Objective:GetDisplayName() .. " -> " .. Objective:GetName() .. ": " .. get_key_for_value(criterias, Objective:GetType()))  -- TODO
```
Comment (tonitch) [inline cmd_scoreboard.lua:221]: I was wondering... is there a better way to send the result of a command (with multiple line) than doing it with `LOG()` ?

**Review by tonitch — COMMENTED** [review]
I think I'm quite done ?!

File: cmd_scoreboard.lua (hunk 1 above)
Comment (mathiascode) [reply cmd_scoreboard.lua]: This is how the /tps command does it: https://github.com/cuberite/Core/blob/baf6a4e7050f8ed3c19dab50924863ed704a3eba/cmd_tps.lua#L34-L42

I don't remember if we have other commands in the new format that output multiple lines.

File: CONTRIBUTORS [hunk 2]
```diff
@@ -28,3 +28,4 @@ Taugeshtu
 tigerw (Tiger Wang)
 tonibm19
 xoft (Mattes Dolak/madmaxoft)
+tonitch (Debucquoy Anthony)
```
Comment (mathiascode) [inline CONTRIBUTORS:31]: Alpha-sort

**Review by mathiascode — COMMENTED** [review]
Use PascalCase in general.

**Comment by tonitch** [comment]
There we go, Ready again!

**Review by tonitch — COMMENTED** [review]
all done

**Comment by mathiascode** [comment]
@tonitch Could you create a follow-up PR addressing some unhandled exceptions when invalid command arguments are provided?

**Pull request merged**